appserver/admingui/payara-console-extras/src/main/resources/requestTracing/requestTracing.jsf

```diff
@@ -189,6 +192,9 @@
                       helpText="$resource{i18nrt.requestTracing.configuration.historicTraceStoreSizeLabelHelpText}">
             <sun:textField id="historicTraceStoreSizeField" columns="$int{40}"  styleClass="integer" maxLength="30" 
                            text="#{pageSession.valueMap['historicTraceStoreSize']}" />
+            <f:verbatim>
+            <div class="ConWrnLblTxt_sun4" style="padding-top: 10px; font-size: 90%;">Please note that from 5.194 onwards the store size refers to the size of a single store shared by all cluster members. Any configuration change regarding the store should be applied to all configurations equally to prevent unbalanced sharing. All affected configuration are extracted and moved to a central configuration in a future release.</div>
```

Comment:
Could I suggest changing the final part of the sentence to "All _affected configurations will be_ extracted and moved to a central configuration in a future release." It's picky but it would just make the tense consistent

Reply:
Changed.